src/domain/event/eventInfo/__tests__/EventInfo.test.tsx

```diff
@@ -206,3 +210,60 @@ it('should create ics file succesfully when end time is not defined', async () =
 
   expect(FileSaver.saveAs).toBeCalled();
 });
+
+it('should show audience age info on signle course page', async () => {
+  render(<EventInfo event={event} eventType="course" />, {
+    routes: [`/fi/courses`],
+  });
+  await actWait();
+
+  expect(screen.queryByText(/5-15 -vuotiaat/i)).toBeInTheDocument();
+});
+
+it('should show formatted audience age info on signle course page if min age is not specified', async () => {
+  render(
+    <EventInfo event={{ ...event, audienceMinAge: null }} eventType="course" />,
+    {
+      routes: [`/fi/courses`],
+    }
+  );
+  await actWait();
+
+  expect(screen.queryByText(/0-15 -vuotiaat/i)).toBeInTheDocument();
+});
+
+it('should show formatted audience age info on signle course page if max age is not specified', async () => {
+  render(
+    <EventInfo event={{ ...event, audienceMaxAge: null }} eventType="course" />,
+    {
+      routes: [`/fi/courses`],
+    }
+  );
+  await actWait();
+
+  expect(screen.queryByText(/5\+ -vuotiaat/i)).toBeInTheDocument();
+});
+
+it('should hide audience age info on single course page if min and max ages are not specified', async () => {
+  render(
+    <EventInfo
+      event={{ ...event, audienceMinAge: null, audienceMaxAge: null }}
+      eventType="course"
+    />,
+    {
+      routes: [`/fi/courses`],
+    }
+  );
+  await actWait();
+
```

Comment:
There could also check that the title is not present: 
expect(screen.queryByText(/Ikäryhmä/i)).not.toBeInTheDocument();

Reply:
done